unnamed_toaster/unnamed_toaster/scripts/match_logic.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+
+"""
+    Copyright (c) 2019-2020 Concord Robotics Inc.
+    All right reserved.
+
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+      * Redistributions of source code must retain the above copyright
+        notice, this list of conditions and the following disclaimer.
+      * Redistributions in binary form must reproduce the above copyright
+        notice, this list of conditions and the following disclaimer in the
+        documentation and/or other materials provided with the distribution.
+      * Neither the name of Vanadium Labs LLC nor the names of its
+        contributors may be used to endorse or promote products derived
+        from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+    ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+    WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+    DISCLAIMED. IN NO EVENT SHALL VANADIUM LABS BE LIABLE FOR ANY DIRECT, INDIRECT,
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+    LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA,
+    OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+    LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+    OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+    ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+"""
+
+import rospy
+
+import tf2_ros
+import tf2_geometry_msgs
+
+from geometry_msgs.msg import PointStamped
+from std_msgs.msg import Float64
+
+from unnamed_toaster.unnamed_toaster.scripts.shooter_math import ShooterMath
```

Comment:
so this isn't going to work - the proper way to do this is to put things you want to import in a "src/unnamed_toaster" folder. That folder needs an __init__.py file, and you need to have a setup.py and uncomment catkin_python_setup() in the CMakeLists.txt

Alternatively, you can just do "from shooter_math import ShooterMath" - but that only works since these are in the same directory.

Reply:
Removed line